docs/gaslessApi.md

```diff
@@ -0,0 +1,91 @@
+---
+id: gasless-api
+title: Gasless API
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+The gas fee is preventing new users step into the web3 world. Users must learn to get the native token (CKB or ETH) before playing a blockchain game or exchanging tokens with a DEX. The gasless feature can provide a way for developers to sponsor transaction fees for users to give them a smooth experience.
+
+The gas feature is based on the [ERC-4337 solution](https://eips.ethereum.org/EIPS/eip-4337) but way simpler. To use a such feature, users sign and send a special gasless transaction to call a specific smart contract named `Entrypoint`, then `Entrypoint` will call another smart contract named `Paymaster` deployed by developers to check if they are willing to pay the gas fee for this transaction. The special gasless transaction must satisfy the requirements:
+
+- `tx.data` must be the call data of calling `Entrypoint`'s `handleOp(UserOperation calldata op)` function, which contains the target contract and the paymaster address.
+- Must set `tx.gasPrice` to 0 (compatible with Metamask api)
+- The `tx.to` must be set to the `Entrypoint` contract.
+- `tx.paymasterAndData` contains a pay master address.
+- `tx.gas` is equals to `op.callGasLimit + op.verificationGasLimit * 3`.
+
+
+### Use cases
+
+Each solving some decentralized problem. WIth gasless feature some of dapps could be improved with solutions like:
+ - DEX/Lending and rest transaction gas payments with project token (descreasing project inflation);
+ - NFT minting and exchange (to lower the entry threshold for the crypto gaming);
+
+
+### How to use gasless transaction for your dapp
+
+Let's say we have the entrypoint contract at `0x9a11f47c0729fc56d9c44c059987d40703249569` and as a game developer, we want to pay the gas fee for some whitelist users, so we wrote a paymaster contract just like [this one](https://github.com/godwokenrises/account-abstraction/blob/gw-gasless/contracts/samples/GaslessDemoPaymaster.sol) and deployed it at `0x6b019795aa36dd19eb4a4d76f3b9a40239b7c19f`.
```

Comment:
`0x9a11f47c0729fc56d9c44c059987d40703249569` since the example address may confuse users that is a real one to use, suggest we replace it with `ENTRYPOINT_CONTRACT_ADDRESS` or something else